core/src/main/res/drawable/filter_dialog_button_background_dark.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@color/dialog_filter_button_active_dark" android:state_checked="true" />
+    <item android:drawable="@color/dialog_filter_button_inactive_dark" />
```

Comment:
Those are only colors, so they can be moved to the `color` directory.

Reply:
```
<?xml version="1.0" encoding="utf-8"?>
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:color="@color/dialog_filter_button_active_dark" android:state_checked="true" />
    <item android:color="@color/dialog_filter_button_inactive_dark" />
</selector>``
```

Reply:
Yeah, something like that and then moving the file to the color directory would be cool.